react/features/conference/components/Conference.web.js

```diff
@@ -41,6 +41,18 @@ const FULL_SCREEN_EVENTS = [
     'fullscreenchange'
 ];
 
+/**
+ * The CSS class to apply to the root element of the conference so CSS can
+ * modify the app layout.
+ *
+ * @private
+ * @type {Object}
+ */
+const LAYOUT_CLASSES = {
+    HORIZONTAL_FILSTRIP: 'horizontal-filmstrip',
+    VERTICAL_FILMSTRIP: 'vertical-filmstrip'
```

Comment:
Are we going to add TILE_FILMSTRIP to this list with another PR?

Reply:
Yeah, TILE_FILMSTRIP will get added with the PR that introduced the tile view css and the resizing.